pkg/function/restore_data.go

```diff
@@ -176,6 +177,10 @@ func (*restoreDataFunc) Exec(ctx context.Context, tp param.TemplateParams, args
 	}
 	// Validate and get optional arguments
 	restorePath, encryptionKey, pod, vols, backupTag, backupID, err := validateAndGetOptArgs(args)
+
+	if pod != "" {
```

Comment:
doesn't this Function also use pod runner?

Reply:
It does, but the pod name set here is different than the one that is generated by pod runner.

Reply:
I would either drop this or rename the key since it's getting overwritten anyway

Reply:
Use a different key in order to tell apart the 2 pod names?